pkg/agent/flannel/setup.go

```diff
@@ -67,6 +72,11 @@ const (
 	"SubnetAddCommand": "read PUBLICKEY; wg set flannel.1 peer $PUBLICKEY endpoint $PUBLIC_IP:51820 allowed-ips $SUBNET persistent-keepalive 25",
 	"SubnetRemoveCommand": "read PUBLICKEY; wg set flannel.1 peer $PUBLICKEY remove"
 }`
+
+	emptyIPv6Network = "::/0"
+
+	ipv4 = iota
```

Comment:
This makes ipv4 = 0 and ipv6 = 1, so the `ipv4 + ipv6` tests below are basically just checking for ipv6? How about:
```suggestion
	_ = iota
	ipv4
```

Reply:
Yes, I also thought about that and in my first commits, I was doing `iota + 1` to avoid 0 and keep 0 as the empty value when there is an error. However, then I saw that when doing dual-stack `netMode=17`. I investigated a bit and noticed that the integer that iota provides depends on the number of const variables, so you get 0 only if it is the first const variable. Therefore, in the current situation, ipv4=7 and ipv6=8.

Reply:
Demonstration:
```
package main

import (
	"fmt"
)

const (
	name = "Manuel"
	age = 34

	hola = iota
	hello
	hallo
)

func main() {
	fmt.Println("Starting")
	fmt.Println(hola)
	fmt.Println(hello)
	fmt.Println(hallo)
}
```
Output:
```
Starting
2
3
4
```

Reply:
Ah great, thanks! For some reason I thought iota started numbering on the first const that uses it. It's an odd language feature that I haven't gotten used to yet.